hilo.rb

```diff
@@ -0,0 +1,26 @@
+random_number = rand(1..100)
+# print random number for testing purposes
+puts random_number
+puts "Enter a number between 1 and 100. press enter to submit your guess"
+guess = $stdin.readline()
+guess_as_int = Integer(guess)
```

Comment:
You can also use `guess.to_i`.  Everything in Ruby is an object. When you run your rails console you can do something like `guess.methods` which will show all public methods available to your object.